kubeflow/pull-upstream.sh

```diff
@@ -16,9 +16,9 @@
 
 set -ex
 
-export KUBEFLOW_MANIFESTS_VERSION=v1.6.0
+export KUBEFLOW_MANIFESTS_VERSION=v1.6.1
 export KUBEFLOW_MANIFESTS_REPO=https://github.com/kubeflow/manifests.git
-export KUBEFLOW_PIPELINES_VERSION=2.0.0-alpha.4
+export KUBEFLOW_PIPELINES_VERSION=2.0.0-alpha.6
```

Comment:
I'm curious does GCP distribution needs to align with generic distribution in terms of binary versions?
I mean `kubeflow/manifests` has released 1.6.1 with KFP 2.0.0-alpha.5. https://github.com/kubeflow/manifests/tree/v1.6.1

Reply:
While we are using pre-released KFP, it shouldn't bring too many issues. Other components are in sync with the platform-agnostic Kubeflow.